frontend/packages/shared/src/api/paths.js

```diff
@@ -32,8 +32,8 @@ export const widgetSettingsPath = (org, app) => `${basePath}/${org}/${app}/app-d
 export const optionListIdsPath = (org, app) => `${basePath}/${org}/${app}/app-development/option-list-ids`; // Get
 export const ruleConfigPath = (org, app, layoutSetName) => `${basePath}/${org}/${app}/app-development/rule-config?${s({ layoutSetName })}`; // Get, Post
 export const datamodelMetadataPath = (org, app, layoutSetName) => `${basePath}/${org}/${app}/app-development/model-metadata?${s({ layoutSetName })}`; // Get
-export const layoutSetPath = (org, app, layoutSetName) => `${basePath}/${org}/${app}/app-development/layout-sets?${s({ layoutSetName })}`; // Put, Post
-export const layoutSetsPath = (org, app) => `${basePath}/${org}/${app}/app-development/layout-sets`; // Get
+export const layoutSetPath = (org, app, layoutSetIdToUpdate) => `${basePath}/${org}/${app}/app-development/layout-sets?${s({ layoutSetIdToUpdate })}`; // Put
+export const layoutSetsPath = (org, app) => `${basePath}/${org}/${app}/app-development/layout-sets?`; // Get
```

Comment:
Har det noe for seg å sette spørsmålstegn på slutten av URL-en her?

Reply:
Ups nei, fjerner det 👍